cdap-app-fabric/src/main/java/io/cdap/cdap/security/TokenSecureStoreRenewer.java

```diff
@@ -143,24 +139,22 @@ public Credentials createCredentials() {
   }
 
   private long calculateUpdateInterval() {
-    return calculateUpdateInterval(yarnConf, secureExplore);
+    return calculateUpdateInterval(yarnConf, secureToken);
   }
 
   /**
    * Calculates the secure token update interval based on the given configurations.
    *
-   * @param cConf the CDAP configuration
    * @param hConf the YARN configuration
    * @return time in millisecond that the secure token should be updated
    */
-  public static long calculateUpdateInterval(CConfiguration cConf, Configuration hConf) {
-    boolean secureExplore = cConf.getBoolean(Constants.Explore.EXPLORE_ENABLED)
-      && UserGroupInformation.isSecurityEnabled();
+  public static long calculateUpdateInterval(Configuration hConf) {
+    boolean secureToken = UserGroupInformation.isSecurityEnabled();
```

Comment:
This is logic change. Previously it was always off when explore is disabled. Now even without explore we do it when security is enabled. @dli357 could you confirm this?

Also you don't need to pass it as parameter, you can get the value on the spot

Reply:
I misinterpreted the && functionality. As explore is disabled now, I have removed secureToken altogether.